.github/workflows/test.yml

```
@@ -14,7 +14,29 @@ jobs:
       - run: |
           npm install
           npm run all
+
+      # Update dist files if there is label dependencies
+      - name: Update dist files
+        if: github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'dependencies')
+        run: |
+          if [[ -z $(git status -s) ]]
+          then
+            echo "No change is required"
+          else
+            echo "Updating dist directory"
+            git config --local user.name "dependabot[bot]"
+            git config --local user.email "49699333+dependabot[bot]@users.noreply.github.com"
```

I think it's better to use the `goreleaserbot <goreleaser@carlosbecker.com>`
```suggestion
            git config --local user.name "goreleaserbot"
            git config --local user.email "goreleaser@carlosbecker.com"
```